### PR TITLE
Re-enable frontend coding style checks in CI

### PR DESCRIPTION
**Pull request by lukasbestle**
### Enhancements
- `k-text` has a new `html` prop to pass content that should be rendered as HTML (instead of providing text via the default slot). Use carefully as rendering HTML can be a gateway for XSS attacks.

### Refactoring

- Our CI now checks frontend coding style again

### Breaking changes

None

### For review team

- [x] Add changes to release notes draft in Notion
- [x] ~Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)~

### CODE REVIEW

File: panel/src/components/Layout/Overlay.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			@mousedown="close"
 		>
 			<k-loader v-if="loading" class="k-overlay-loader" />
-			<slot v-else :close="close" :isOpen="isOpen" />
+			<slot v-else :close="close" :is-open="isOpen" />
```
Comment (lukasbestle) [inline panel/src/components/Layout/Overlay.vue:15]: Not sure what this change does TBH

**Comment by lukasbestle** [comment]
There are two sets of errors still left:

- **Using v-html on component may break component's content** (https://github.com/vuejs/eslint-plugin-vue/issues/1724)
  These `v-html` uses may actually be an issue and we need to verify each one
- **Prop "value" should define at least its type** for `TableCell.vue`
  This prop is explicitly mixed; is there a way to define a mixed type in Vue or do we need to silence that error?

File: panel/src/components/Layout/Overlay.vue (hunk 1 above)
Comment (johannschopplich) [reply panel/src/components/Layout/Overlay.vue:15]: [vue/attribute-hyphenation](https://eslint.vuejs.org/rules/attribute-hyphenation.html) requires the props to be hyphenated.

**Comment by distantnative** [comment]
Regarding `vue/no-v-text-v-html-on-component`, all usages are on `k-text` which is fine as it is a single div-container anyways filled by the slot, so filling it with HTML doesn't break anything. We could add an `html` prop to `k-text` though and then use `v-html` in `Text.vue`, minimising it to one mention in our code.

Regarding `vue/require-prop-types`, we might just define all possible types then: String, Number
Boolean, Array, Object, Date, Function, Symbol. However, this could lead to issues maybe if a certain value is casted to a specific type because the order we mention them (not sure if that's really a case). Could be an argument for just silencing this one.

**Comment by distantnative** [comment]
Like this https://github.com/getkirby/kirby/commit/ad4c97455b8c16286486c76c8b093aeb8296b44b

**Comment by lukasbestle** [comment]
Both sounds great! 👍

> Could be an argument for just silencing this one.

I agree.

File: panel/src/components/Layout/Overlay.vue (hunk 1 above)
Comment (lukasbestle) [reply panel/src/components/Layout/Overlay.vue:15]: Yes, I understand that part. But I was wondering if the different syntax could break behavior.

Comment (johannschopplich) [reply panel/src/components/Layout/Overlay.vue:15]: It shouldn't. As I understand it, the Vue compiler will handle pascal case from props and attribute case in templates. The compiler is case-tolerant, that's why it's working currently as well.

**Review by distantnative — APPROVED** [review]

**Pull request merged**